pkg/acceptance/testdata/node/base-test.js

```diff
@@ -38,11 +38,6 @@ describe('select', () => {
 
 describe('error cases', () => {
   const cases = [{
-    name: 'not enough params',
```

Comment:
did you confirm that postgres does not expect this behavior?

Reply:
Added a test [here](https://github.com/cockroachdb/cockroach/pull/87535/commits/4e42e72f4bc7f87b8e6916a7d8b5b293899d0bea#diff-0c2048a8b4d6014b3c805c033e13e0cd30ea5dfb36a3db85d37d34bcc84c4f47R30-R44) to verify.